easybuild/easyconfigs/m/MSM/MSM-1.0-intel-2017b.eb

```diff
@@ -0,0 +1,34 @@
+name = "MSM"
+version = "1.0"
+
+homepage = "https://github.com/ecr05/MSM_HOCR"
+description = """Multimodal Surface Matching with Higher order Clique Reduction"""
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+
+source_urls = ['https://github.com/ecr05/MSM_HOCR/archive']
+sources = ['%(version)s.tar.gz']
+patches = ['MSM-1.0_Fix_abs_overload.patch']
+checksums = [
+    'ed6e4b29913be2dfe9694edbe6181dce07407a8b8d2cafbc0859b834074370a7',
+    '5ec37ed93f30eb1fd0bd34038815733658e42a3a114201e80a1d480a8bcdb623'  # MSM-1.0_Fix_abs_overload.patch
+]
+
+dependencies = [
+    ('FSL', '5.0.10'),
+    ('zlib', '1.2.11'),
+]
+
+_binfiles = [
+    'estimate_metric_distortion', 'msm', 'msmapplywarp',
+    'msm_metric_sim', 'msmresample', 'surfconvert'
+]
+
+files_to_copy = [(['src/MSM/' + f for f in _binfiles], 'bin')]
+
+sanity_check_paths = {
+    'files': ['bin/' + f for f in _binfiles],
+    'dirs': ['doc', 'include', 'lib']
+}
```

Comment:
@vanzod Why isn't the `files_to_copy` and `sanity_check_paths` part not handled by the custom easyblock for `MSM`?